x/nexus/keeper/rate_limit_test.go

```diff
@@ -44,7 +45,7 @@ func TestSetRateLimit(t *testing.T) {
 	})
 
 	setRateLimitFails := func(msg string) ThenStatement {
-		return Then("set rate limit will fail", func(t *testing.T) {
+		return Then(fmt.Sprintf("set rate limit will fail %s", msg), func(t *testing.T) {
```

Comment:
the resulting message doesn't make sense,  eg "... then set rate limit will fail not a registered chain"